Modules/Tracker/QuestieTrackerUtils.lua

```diff
@@ -17,19 +17,25 @@ local tinsert = table.insert
 function QuestieTracker.utils:ShowQuestLog(quest)
     -- Priority order first check if addon exist otherwise default to original
     local questFrame = QuestLogExFrame or ClassicQuestLog or QuestLogFrame;
-    HideUIPanel(questFrame);
+    --HideUIPanel(questFrame); -- don't use as I don't see why to use and protected function taints in combat
     local questLogIndex = GetQuestLogIndexByID(quest.Id);
     SelectQuestLogEntry(questLogIndex)
 
     -- Scroll to the quest in the quest log
     local scrollSteps = QuestLogListScrollFrame.ScrollBar:GetValueStep()
     QuestLogListScrollFrame.ScrollBar:SetValue(questLogIndex * scrollSteps - scrollSteps * 3);
 
-    ShowUIPanel(questFrame);
+    if not questFrame:IsShown() then
+        if not InCombatLockdown() then
+            ShowUIPanel(questFrame)
 
-    --Addon specific behaviors
-    if(QuestLogEx) then
-        QuestLogEx:Maximize();
+            --Addon specific behaviors
+            if(QuestLogEx) then
+                QuestLogEx:Maximize();
+            end
+        else
+            Questie:Print("Can't open Quest Log while in combat. Open it manually.")
```

Comment:
I forgot to addon translations for this. Also I'm not yet sure how translations work.

Reply:
Oh right. You basically just add a table entry into [the best fitting file](https://github.com/Questie/Questie/blob/master/Localization/Translations/DebugMessages.lua) and add all the translations you can. For unknown translations add `false`.